mailstream/lang/C/messages.po

```diff
@@ -1,20 +1,20 @@
-# ADDON mailstream
-# Copyright (C) 
-# This file is distributed under the same license as the Friendica mailstream addon package.
-# 
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```

Comment:
This doesn't look right, please revert the changes to these lines.

Reply:
Done.